docs/en/advanced/Alluxio-Storage-Management.md

```diff
@@ -182,71 +210,149 @@ For example, to turn off asynchronous eviction:
+worker and then kicks off the eviction process to free up enough space to serve that request. This
+leads to many small eviction attempts, which is less efficient but maximizes the utilization of
+available Alluxio space. In general, we recommend to use asynchronous eviction.
 
 ### Eviction Policies
 
-TODO(Zac): cleanup this section.
+<!-- ################## -->
+<!-- TODO(Zac): cleanup this section.
 We should describe the default behavior, and give use cases for why users would want to use each evictor.
-And we mention that users can write their own evictors, but don't give any resources for them to do that.
-
+And we mention that users can write their own evictors, but don't give any resources for them to do that. -->
+<!-- ################## -->
 
-Alluxio uses evictors for deciding which blocks to evict, when space needs to be
-freed. Users can specify the Alluxio evictor to achieve fine grained control over the eviction
+Alluxio uses evictors for deciding which blocks to remove from Alluxio storage, when space needs to
+be freed. Users can specify the Alluxio evictor to achieve fine grained control over the eviction
 process.
 
-Alluxio supports custom evictors. Out-of-the-box implementations include:
-
-* **GreedyEvictor**: Evicts arbitrary blocks until the required size is freed.
-* **LRUEvictor**: Evicts the least-recently-used blocks until the required size is freed.
-* **LRFUEvictor**: Evicts blocks based on least-recently-used and least-frequently-used with a configurable weight.
-    If the weight is completely biased toward least-recently-used, the behavior will be the same as
-    the LRUEvictor.
-* **PartialLRUEvictor** : Evicts based on least-recently-used but will choose StorageDir with
+Out-of-the-box evictor implementations include:
+
+- **GreedyEvictor**: Evicts arbitrary blocks until the required size is freed.
+- **LRUEvictor**: Evicts the least-recently-used blocks until the required size is freed.
+**This is Alluxio's default evictor**
+- **LRFUEvictor**: Evicts blocks based on least-recently-used and least-frequently-used with a
+configurable weight.
+    - If the weight is completely biased toward least-recently-used, the behavior will be the same
+    as the LRUEvictor.
+    - The applicable configuration properties are `alluxio.worker.evictor.lrfu.step.factor` and
+    `alluxio.worker.evictor.lrfu.attenuation.factor.` which can be found in
+    [configuration properties]({{site.baseurl}}{% link en/reference/Properties-List.md %}#alluxio.worker.evictor.lrfu.step.factor)
+- **PartialLRUEvictor** : Evicts based on least-recently-used but will choose StorageDir with
 maximum free space and only evict from that StorageDir.
 
-In the future, additional evictors will be available. Since Alluxio supports custom evictors,
-you can also develop your own evictor appropriate for your workload.
+The evictor utilized by workers is determined by the Alluxio property
```

Comment:
The available evictor class named are the bolded names in the list. Maybe a sentence to let the readers know that, or rearrange the order so we could have something like "the available classes are below:"

Reply:
The configuration property requires a fully qualified classname, so instead of `GreedyEvictor`, users need to put `alluxio.worker.block.evictor.GreedyEvictor`.